### PR TITLE
Embed QSQLITE driver at Amlogic platform

### PR DESCRIPTION
**Pull request by Portisch**
**1.** Tell us something about your changes.
Embed QSQLITE driver on Amlogic platform into hyperiond binary by `Q_IMPORT_PLUGIN`

**2.** If this changes affect the .conf file. Please provide the changed section
none

**3.** Reference an issue (optional)
QSQLITE is not part of the system.
I am not sure if this would be needed on other platforms too.

Note: For further discussions use our forum: forum.hyperion-project.org

### CODE REVIEW

File: libsrc/db/DBManager.cpp [hunk 1]
```diff
@@ -8,6 +11,11 @@
 #include <QUuid>
 #include <QDir>
 
+#ifdef ENABLE_AMLOGIC
```
Comment (Paulchen-Panther) [inline libsrc/db/DBManager.cpp]: Please change to #if ENABLE_AMLOGIC.
The variable ENABLE_AMLOGIC exists in any case and should only be checked for its value.

Comment (Portisch) [reply libsrc/db/DBManager.cpp]: Looks like this is not working without a compare.
So how to handle this? There are some more #ifdef like I used:
https://github.com/hyperion-project/hyperion.ng/blob/master/libsrc/grabber/framebuffer/FramebufferFrameGrabber.cpp#L49

**Comment by Portisch** [comment]
I just checked it again. It have to be `#ifdef ENABLE_AMLOGIC`, When `ENABLE_AMLOGIC = 1` is set it's defined. If `ENABLE_AMLOGIC = 0` `ENABLE_AMLOGIC` is undefined.

**Comment by Portisch** [comment]
Why is for `Build Debian Stretch (AMD64) packages` `ENABLE_AMLOGIC = ON`?
Same for `Build Debian Stretch (i386) packages`
qt-everywhere have to be build with `-qt-sqlite` to add the QSQLITE plugin

**Comment by Portisch** [comment]
I think best is to include this as patch in our local build - closed!

**Pull request closed without merging**